app/controllers/comments_controller.rb

```diff
@@ -17,11 +17,12 @@ def create
 
     respond_to do |format|
       if @comment.save
```

Comment:
- [ ] Your create method should be similar to this
`def create`
    `@comment = @current_user.comments.new(params)`
    `@comment.post_id = params[:post_id]`

    `if @comment.save`
      `redirect_to post_path(@comment.post)`
    `else`
      `@article = Post.find(params[:post_id])`
    `  redirect_to post_path(@post)`
    `end`
`  end`
rather than what you have with `html` incorporated

Reply:
@t-yanick Thanks so much for your feed back.